pages/implementation/avo-and-git.mdx

```diff
@@ -52,20 +40,49 @@ _Onboarding flow event in Avo, sent from iOS_
 _Property always sent from iOS, never from Android_
 </center>
 
-When the tracking plan changes made on the branch have been <Link passHref href="/data-design/day-to-day-workflow#a-namestep-4-ask-for-reviewa-step-4--ask-for-review"><a>peer-reviewed and approved</a></Link> in Avo, they can start the tracking implementation. Using <Link passHref href="/implementation/cli#using-avo-branches"><a>the Avo CLI</a></Link> the iOS developer can pull their type-safe generated Avo code based on the tracking plan from the Avo branch, into their feature branch on git.
+#### Branched tracking plan implementation
+
+When the tracking plan changes made on the branch have been <Link passHref href="/data-design/day-to-day-workflow#a-namestep-4-ask-for-reviewa-step-4--ask-for-review"><a>peer-reviewed and approved</a></Link> in Avo, they can start the tracking implementation.
+
+Using <Link passHref href="/implementation/cli#using-avo-branches"><a>the Avo CLI</a></Link> the iOS developer can pull their type-safe generated Avo code based on the tracking plan from the Avo branch, into their feature branch on git: `avo pull new-onboarding-events` where `new-onboarding-events` is the Avo branch containing the Tracking Plan changes. By doing so, they've pinned the `avo.json` file in their project to the current state of the Tracking Plan on that Avo branch.
+
+> `avo status` uses the Tracking Plan version defined in `avo.json`to verify everything is implemented. That means `avo status` doesn't break if changes are made to your Tracking Plan that haven't been pulled into code.
+
+Once the tracking has been implemented, and the feature is ready to go, the developer gets their git branch merged like they usually would. As soon as the git branch is merged into the git main branch, they merge their Avo branch in the Avo main branch. By doing so, the main branch in Avo always represents the current state of the Avo tracking plan, while the tracking plan implementation stays pinned to the branch we just implemented. The `avo.json` file still points to the head of the Avo branch they just implemented, since that's the version of the Tracking Plan the most recent tracking implementation was based on.
 
-Once the tracking has been implemented, and the feature is ready to go, the developer gets their git branch merged like they usually would. As soon as the git branch is merged into the git main branch, they merge their Avo branch in the Avo main branch. By doing so, the Avo main branch always represents the current state of Avo tracking on the git main branch on every platform.
+When it’s time for the Android team to implement the same changes, they would go through the same flow, but instead of creating the events and properties in Avo, they would only need to add the Android source to the events they want to implement, set the properties to be always sent from Android and run `avo pull` from their Avo branch to implement the tracking changes.
```

Comment:
Do they need to explicitly set the properties to be always sent on Android, won't that happen automatically?

Reply:
We could potentially skip this detail here, but earlier in the guide they mark the property as sent from iOS and not Android